File: app/src/main/java/com/callmangement/EHR/models/SurveyFormDocInfo.java
package com.callmangement.EHR.models;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

public class SurveyFormDocInfo implements Serializable {

    @SerializedName("documentPath")
    private String documentPath;

    @SerializedName("documentName")
    private String documentName;

    public String getDocumentPath() {
        return documentPath;
    }

    public void setDocumentPath(String documentPath) {
        this.documentPath = documentPath;
    }

    public String getDocumentName() {
        return documentName;
    }

    public void setDocumentName(String documentName) {
        this.documentName = documentName;
    }
}

File: app/src/main/java/com/callmangement/ehr/models/SurveyFormDocInfo.kt
package com.callmangement.ehr.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable

class SurveyFormDocInfo : Serializable {
    @JvmField
    @SerializedName("documentPath")
    var documentPath: String? = null

    @SerializedName("documentName")
    var documentName: String? = null
}

